test_runner/fixtures/neon_fixtures.py

```diff
@@ -428,6 +428,7 @@ def __init__(
         preserve_database_files: bool = False,
         initial_tenant: Optional[TenantId] = None,
         initial_timeline: Optional[TimelineId] = None,
+        enable_generations: bool = False,
```

Comment:
This is unsed; usage of `enable_generations` in the tests you added suggests this arg to the constructor should be removed.